content/en/docs/concepts/workloads/controllers/job.md

```diff
@@ -374,6 +375,107 @@ Here, `W` is the number of work items.
 
 ## Advanced usage
 
+### Suspending a Job
+
+{{< feature-state for_k8s_version="v1.21" state="alpha" >}}
+
+{{< note >}}
+Suspending Jobs is available in Kubernetes versions 1.21 and above. You must
+enable the `SuspendJob` [feature gate](/docs/reference/command-line-tools-reference/feature-gates/)
```

Comment:
If behind the feature gate, you need to add it to Alpha/Beta Feature gates table https://github.com/kubernetes/website/blob/master/content/en/docs/reference/command-line-tools-reference/feature-gates.md#feature-gates-for-alpha-or-beta-features

Reply:
thanks for catching that, I've added row

Reply:
could you make sure that you also have a description for the feature https://github.com/kubernetes/website/blob/master/content/en/docs/reference/command-line-tools-reference/feature-gates.md#list-of-feature-gates-feature-gates?

Reply:
Just caught that a second after I pushed :) fixed.